CHANGELOG.md

```diff
@@ -1,5 +1,18 @@
 # Quarkus Tools for Visual Studio Code Changelog
 
+## 1.5.0 (April 30, 2020)
+
+### Enhancements
+
+ * Code snippets for MicroProfile fault tolerance annotations. See [LS#307](https://github.com/redhat-developer/quarkus-ls/issues/307)
+ * Support for MicroProfile Context Propagation properties. See [LS#242](https://github.com/redhat-developer/quarkus-ls/issues/242)
+ * Create Container Image snippets, update Quarkus datasource snippets. See [#227](https://github.com/redhat-developer/vscode-quarkus/issues/227)
```

Comment:
The PR for that issue is not merged yet

Reply:
It'll become "Update Quarkus datasource snippet" given https://github.com/redhat-developer/quarkus-ls/pull/310#discussion_r417490326